src/main/java/DBDriverWrapper.java

```diff
@@ -0,0 +1,13 @@
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+public interface DBDriverWrapper {
+
+  Connection connect(String host, String database, String userName, String password);
```

Comment:
This is only being used internally, probably should not expose in the interface either